resources/views/store/auth/login.blade.php

```diff
@@ -17,10 +17,15 @@
         <div class="login-container">
             <h2>Log In</h2>
             @if ($errors->has('error_message'))
-                <div class="alert alert-danger">
+                <div class="alert alert-warning">
                     <strong>{{ $errors->first('error_message') }}</strong>
                 </div>
             @endif
+            @if (session('status'))
```

Comment:
What messages are you seeing in status? Just curious.

Reply:
Auth returns "Your password has been reset!"

Reply:
Awesome.